src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -1140,6 +1140,87 @@ static class Test { static void Method() { int a = MyStruct.Some$$Field; } }";
                 Documentation("My Field"));
         }
 
+        [WorkItem(538638, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/538638")]
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public async Task TestPartialMethodDocComment_01()
+        {
+            var markup =
+@"partial class MyClass
+{
+    ///<summary>My Method Definition</summary>
+    public partial void MyMethod();
```

Comment:
any concern that all thse tests test the public case?  i know ublic partials are different from nonpublic ones.  would testing that be useful?  or is all the same to the compiler?

Reply:
> or is all the same to the compiler?

I think it is, but I added one test where the partial method has no access modifiers for good measure.